### PR TITLE
Version bumps

### PR DESCRIPTION
**Pull request by bgamari**
This bumps a variety of bounds for GHC 8.8.

### CODE REVIEW

File: patches/exceptions-0.10.0.cabal [hunk 1]
```diff
@@ -36,7 +36,7 @@ library
   build-depends:
     base                       >= 4.3      && < 5,
     stm                        >= 2.2      && < 3,
-    template-haskell           >= 2.2      && < 2.15,
+    template-haskell           >= 2.2      && < 2.16,
```
Comment (RyanGlScott) [inline patches/exceptions-0.10.0.cabal]: It's worth noting that there's now a more recent version of `exceptions` on Hackage (see also #112), so this patch will need to be updated to be against `exceptions-0.10.1`.

File: patches/singleton-bool-0.1.4.patch [hunk 2]
```diff
@@ -1,9 +1,3 @@
-commit daf73dfbc836348088d2ad88ecf89bdede000982
-Author: Ryan Scott <ryan.gl.scott@gmail.com>
-Date:   Thu Feb 21 06:51:44 2019 -0500
-
-    Allow building with GHC 8.9
-
```
Comment (RyanGlScott) [inline patches/singleton-bool-0.1.4.patch]: What is the purpose of this change?

**Comment by RyanGlScott** [comment]
This now conflicts with the latest `master`.

**Comment by phadej** [comment]
`head.hackage` upstream repository will be moving to GitLab  https://gitlab.haskell.org/ghc/head.hackage

Announced at https://mail.haskell.org/pipermail/ghc-devs/2019-July/017934.html

---

I'm closing this PR here, please reopen it on GitLab if it's still relevant.
